src/bank.ts

```diff
@@ -402,6 +402,11 @@ export class Bank implements RestServer, Initializable {
     if (!networkInitiated && !user.admin && user.balance < details.amount) {
       throw new ErrorWithStatusCode(402, "Insufficient funds");
     }
+    if (forceAmountToZero) {
+      details.amount = 0.000001;
+    }
+    const totalAmount = forceAmountToZero ? 0.000001 : details.amount;
+
```

Comment:
Redundant check, no?
const totalAmount = details.amount;
will do the same.

Reply:
Yes.  Side-effect of multiple passes of editing.